build_tools/github/build_wheels.sh

```diff
@@ -8,11 +8,23 @@ if [[ "$RUNNER_OS" == "macOS" ]]; then
     # Make sure to use a libomp version binary compatible with the oldest
     # supported version of the macos SDK as libomp will be vendored into the
     # scikit-learn wheels for macos. The list of binaries are in
-    # https://packages.macports.org/libomp/.  Currently, the oldest
-    # supported macos version is: High Sierra / 10.13. When upgrading this, be
-    # sure to update the MACOSX_DEPLOYMENT_TARGET environment variable in
-    # wheels.yml accordingly. Note that Darwin_17 == High Sierra / 10.13.
-    wget https://packages.macports.org/libomp/libomp-11.0.1_0+universal.darwin_17.i386-x86_64.tbz2 -O libomp.tbz2
+    # https://packages.macports.org/libomp/.
+    if [[ "$CIBW_BUILD" == *-macosx_arm64 ]]; then
+        # arm64 builds must cross compile because CI is on x64
+        export PYTHON_CROSSENV=1
+        # SciPy requires 12.0 on arm to prevent kernel panics
+        # https://github.com/scipy/scipy/issues/14688
+        # We use the same deployment target to match SciPy.
+        export MACOSX_DEPLOYMENT_TARGET=12.0
+        wget https://packages.macports.org/libomp/libomp-11.0.1_0.darwin_20.arm64.tbz2 -O libomp.tbz2
+    else
+        # Currently, the oldest supported macos version is: High Sierra / 10.13.
+        # When upgrading this, be sure to update the MACOSX_DEPLOYMENT_TARGET
+        # environment variable in wheels.yml accordingly. Note that Darwin_17 ==
+        # High Sierra / 10.13.
+        export MACOSX_DEPLOYMENT_TARGET=10.13
+        wget https://packages.macports.org/libomp/libomp-11.0.1_0+universal.darwin_17.i386-x86_64.tbz2 -O libomp.tbz2
```

Comment:
Just was working on this for scikit-image as well and noticed there were no `universal2` libomp wheels on macports. It was [suggested in the napari zulip](https://napari.zulipchat.com/#narrow/stream/212875-general/topic/testing.20skimage.20on.20M1) that we could potentially use `lipo` to make a universal2 `libomp` from the separate `x86_64` and `arm64` ones, but perhaps it is not worth the effort at this point?

Reply:
I do not think it's worth the effort for now. I plan to revisit if an issue opens up and there is demand for a `universal2` wheel.

Reply:
also, I'm not sure about needing to bump to 12.0 for the deployment target. I had left this at 11.0 on scikit-image and the preliminary wheels we built pass the test suite when they were tested. Did you encounter failures for scikit-learn with 11.0?

Reply:
I guess given that we have SciPy as a required dependency and it only provides 12.0 wheels we should perhaps do the same.

Reply:
> I guess given that we have SciPy as a required dependency and it only provides 12.0 wheels we should perhaps do the same.

Yes, I don't see how it would be possible to install scikit-learn or scikit-image wheels on macos 11 if scipy only provides wheels for macos 12+ anyways. Better be consistent in the stack.